src/gluonts/zebras/_split_frame.py

```diff
@@ -42,6 +43,14 @@ class SplitFrame:
     default_tdim: int = -1
     _pad: Pad = Pad()
 
+    def __post_init__(self):
+        for column in itertools.chain(self._past, self._future):
+            self.tdims.setdefault(column, self.default_tdim)
+
+        # this triggers checks for past_length and future_length
+        self.past
+        self.future
```

Comment:
I don't know if this is how it's generally done. Would something like this be more readable:
```py
_, _ = self.past, self.future
```

Reply:
Maybe, I'm not sure. But let's do this.